.github/workflows/src/summarize-checks/summarize-checks.js

```diff
@@ -566,8 +565,6 @@ export function updateLabels(existingLabels, impactAssessment) {
   };
 
   if (impactAssessment) {
-    console.log(`Downloaded impact assessment: ${JSON.stringify(impactAssessment)}`);
```

Comment:
dupe of earlier log in summarizeChecksImpl

Reply:
I just merged a PR that removed the earlier output.

Reply:
I still see it logged twice in main:

https://github.com/Azure/azure-rest-api-specs/blob/24bd27a1671c4a4575cac18cd0adacf498bda616/.github/workflows/src/summarize-checks/summarize-checks.js#L374

https://github.com/Azure/azure-rest-api-specs/blob/24bd27a1671c4a4575cac18cd0adacf498bda616/.github/workflows/src/summarize-checks/summarize-checks.js#L553